Author: Feignint <37488595+FeignInt@users.noreply.github.com>
Subject: explain where AUTHORS file can be found in sdlmain
Last-Update: 2020-05-17
--- a/src/gui/sdlmain.cpp
+++ b/src/gui/sdlmain.cpp
@@ -369,7 +369,7 @@ doing so.  There is NO WARRANTY, to the extent permitted by law.
 
 This program (dosbox-staging) is modified version of DOSBox.
 Copyright (C) 2020 The DOSBox Team, published under GNU GPLv2+
-Read AUTHORS file for more details.
+For more details see /usr/share/doc/dosbox-staging/AUTHORS
 )";
 
 #if C_OPENGL
